src/components/Search/SearchBar.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export default function SearchBar({ query, setQuery, callback }) {
```

Comment:
It looks like your onChange is working and the onClick isn't being utilized here.